### PR TITLE
Like

### PR DESCRIPTION
**Pull request by Megha-Joshi**
This PR includes -
- Add video card to like Page
- Remove video card from like Page
- Toggle like and unlike button from home page.

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
small PR
folder structure is proper
code looks good to merge

File: src/context/videoContext.js [hunk 1]
```diff
@@ -24,6 +26,18 @@ const VideoProvider = ({children}) => {
...
+                return{
+                    ...videoState,
+                    likedVideos: action.payload,
+                }
+
+            case "REMOVE_LIKES":
+                return{
+                    ...videoState,
+                    likedVideos: action.payload,
+                }
```
Comment (sudipkundu999) [inline src/context/videoContext.js:40]: you don't need 3 cases doing exact same thing. Refactor logic!

**Review by sudipkundu999 — APPROVED** [review]
Good to merge with code refactor suggestion

File: src/context/videoContext.js (hunk 1 above)
Comment (Megha-Joshi) [reply src/context/videoContext.js:40]: Okay, I'll refactor it.

**Pull request merged**